exercises/practice/alphametics/runtests.jl

```diff
@@ -0,0 +1,102 @@
+using Test
+
+include("alphametics.jl")
+
+@testset "puzzle with three letters" begin
+    @test solve("I + BB == ILL") == Dict(
+        'I' => 1,
+        'B' => 9,
+        'L' => 0
```

Comment:
Might be better to change these to `Symbol`s but I'm not sure

Reply:
Chars are better here, imo